pkg/jetstream/apis/jetstream/v1beta2/consumertypes.go

```diff
@@ -45,6 +45,11 @@ type ConsumerSpec struct {
 	StreamName        string   `json:"streamName"`
 	TLS               TLS      `json:"tls"`
 	Account           string   `json:"account"`
+	HeadersOnly       bool     `json:"headers_only"`
+	MaxRequestBatch   int      `json:"max_batch"`
+	MaxRequestExpires string   `json:"max_expires"`
+	MaxWaiting        int      `json:"max_waiting"`
+	BackOff           []string `json:"backoff"`
```

Comment:
`BackOff` is of time `[]time.Duration`. It is ok to map it in kube with `[]string`?